src/utils/cn.ts

```diff
@@ -0,0 +1,6 @@
+import { clsx, type ClassValue } from "clsx";
+import { twMerge } from "tailwind-merge";
```

Comment:
Please fix this error

![image](https://github.com/user-attachments/assets/9375bf56-6f2b-414b-9855-f67fe44dc389)